.github/workflows/coding-standards.yml

```diff
@@ -20,7 +20,10 @@ jobs:
                 uses: shivammathur/setup-php@v2
                 with:
                     php-version: '8.1'
-                    tools: cs2pr, phpcs
+                    tools: cs2pr, phpcs, php-cs-fixer
 
             -   name: Run PHPCS
                 run: phpcs --standard=PSR1,PSR12 --extensions=php,dist --report=checkstyle -q module config | cs2pr --graceful-warnings
+
+            -   name: Run php-cs-fixer
+                run: php-cs-fixer fix --dry-run --format=checkstyle | cs2pr
```

Comment:
I wanted to comment on this in #186, but I will do it here instead because it is more related. Is this really something we want in an action/in the same action. I feel like that this should go in the `Makefile` such that you can do it locally.

Reply:
Was removed